### PR TITLE
#158628184 Get All rides

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
#### What does this PR do?
Add endpoint for retrieving all available ride offers

#### Description of Task to be completed?
- Add view function to `GET /rides`
- Implement actual ride retrieval within the view function

#### How should this be manually tested?
using your terminal,
- clone this repo
- `cd ride-my-way`
- set and activate your virtual environment
- `git checkout ft-get-rides-158628184`
- Run `python3 run.py`
- Using Postman, access `GET your-server/api/v1/rides`

**screenshot**
all rides

**Pivotal tracker stories **
[Get all ride offers #158628184](https://www.pivotaltracker.com/story/show/158628184)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+venv/*
+__pycache__*
+.coverage
```
[the remark this answers is not in the document]
Comment (meshack-mbuvi) [reply .gitignore]: Done

**Comment by micahoriaso** [comment]
Nice work @meshack-mbuvi

**Comment by JoshuaOndieki** [comment]
Looks good

**Comment by sylsirlym** [comment]
Great Job

**Pull request closed without merging**